extension/fts/test/test_files/stopwords.test

```diff
@@ -49,8 +49,12 @@ alice|0.457137
 -STATEMENT EXPORT DATABASE '${KUZU_EXPORT_DB_DIRECTORY}_fts/stopwords_export_test'
 ---- ok
 -RELOADDB
+# TODO(Ziyi): Fix import database here. RELOADDB is not the correct way to test export/import. The query result is incorrect after import.
```

Comment:
@acquamarin I think there is a bug with import/export fts. Can you double check?